main/src/main.ts

```diff
@@ -182,6 +182,17 @@ export function createWindow(
         width: windowState.bounds.width,
     })
 
+    if (process.platform.includes("darwin")) {
+        const pressAndHold: boolean = systemPreferences.getUserDefault(
+            "ApplePressAndHoldEnabled",
+            "boolean",
+        )
+
+        if (!pressAndHold) {
+            systemPreferences.setUserDefault("ApplePressAndHoldEnabled", "boolean", "true")
```

Comment:
This should be `false` not `"true"`, since it needs to be a `bool` and it should be setting this value to false.

Reply:
yeah that bit is quite confusing as the type for it from electron and the docs state the argument is a string got stuck on that for a while, seems like an issue for electron and their types for this function